docs/docsite/rst/playbooks_conditionals.rst

```diff
@@ -6,11 +6,10 @@ Conditionals
 
 Often the result of a play may depend on the value of a variable, fact (something learned about the remote system), or previous task result.
 In some cases, the values of variables may depend on other variables.
-Further, additional groups can be created to manage hosts based on whether the hosts match other criteria.
-There are many options to control execution flow in Ansible.
-More examples of supported conditionals can be located here: http://jinja.pocoo.org/docs/dev/templates/#comparisons
+Further, additional groups can be created to manage hosts based on whether the hosts match other criteria. This topic covers how conditionals are used in playbooks. 
+
+.. note:: There are many options to control execution flow in Ansible. More examples of supported conditionals can be located here: http://jinja.pocoo.org/docs/dev/templates/#comparisons.
 
```

Comment:
The font could be tricking me but I think the second line needs to be indented here.  Remember that rst uses indentation to denote blocks.  Since the second line is part of the note, it needs to be indented.

Reply:
This should work.

Reply:
Ugh, my bad, it's not the font but github that's tricking me.  That's all one long line.  No changes needed here.